Library/DiscUtils.Core/Internal/Utilities.cs

```diff
@@ -281,6 +281,9 @@ public static string ResolveRelativePath(string basePath, string relativePath)
                 return relativePath;
             }
 
+            if (!basePath.EndsWith(@"\"))
+                basePath = Path.GetDirectoryName(basePath);
+
```

Comment:
I don't think we can do this. On Linux and Windows, `Path.*` might give different results. An example is `GetFileName`.

On Windows, for a Windows path, it will return a file name as expected. But on linux, a Windows path will give you the whole path, and not just the name. The reason is the directory seperators which for Windows contain both variants ("\" and "/") - on linux, only "/" is present.